cheevos/repository_test.go

```diff
@@ -11,6 +11,12 @@ import (
 	"github.com/haleyrc/cheevos/roster"
 )
 
+func TestRepositoryCanBeUsedInTheService(t *testing.T) {
+	_ = &cheevos.Service{
+		Repo: &cheevos.Repository{},
+	}
+}
+
```

Comment:
This can be done more simply with an underscore variable in the package. I'm pretty sure that gets compiled out as well, so it makes no difference either way.